tinytroupe/environment/tiny_world.py

```
@@ -30,7 +27,8 @@ class TinyWorld:
     communication_display = True
 
     def __init__(self, name: str="A TinyWorld", agents=[], 
-                 initial_datetime=datetime.datetime.now(),
+                 initial_datetime=datetime.now(),
```

Ruff (0.8.2)

Do not perform function call `datetime.now` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)